src/streamlink/plugins/filmon.py

```diff
@@ -165,7 +180,19 @@ def channel(self, channel):
 
     def vod(self, vod_id):
         res = self.session.http.get(self.vod_url.format(vod_id))
-        return self.session.http.json(res, schema=self.api_schema)
+        return self.session.http.json(res, schema=self.vod_schema)
+
+
+class TLSAdapter(adapters.HTTPAdapter):
+    def init_poolmanager(self, *args, **kwargs):
+        ctx = ssl.create_default_context()
+        ctx.set_ciphers("DEFAULT:@SECLEVEL=1")
+
+        return super().init_poolmanager(
+            *args,
+            **kwargs,
+            ssl_context=ctx
+        )
```

Comment:
Let's call this `TLSSecLevel1Adapter` and let's move this to `plugin.api.http_session`, so that it can be imported by other plugins as well.

Is the keyword-arg-after-double-star-syntax supported on all python versions? I just mindlessly wrote this. Otherwise kwargs needs to be updated.

Reply:
It seems like the test problem with SSL  is specific to the GH Windows 2019 environment and Python 3.6.8 only.

I don't know the answer to this and unless somebody can (be bothered to) figure it out, perhaps it's easier not to worry about implementing the custom TLSAdaptor until Streamlink drops Python 3.6?

The SSL issue with FilmOn has existed for quite some time now, so putting off this fix for a bit doesn't seem like too big a deal in my opinion.

I would propose we go back to https://github.com/streamlink/streamlink/commit/e6c352d3872f2077a5010c0c7cb93097444ec193 for the time being and merge that.  Additional changes can be made in the future.

Reply:
> perhaps it's easier not to worry about implementing the custom TLSAdaptor until Streamlink drops Python 3.6?
> putting off this fix for a bit doesn't seem like too big a deal in my opinion

That would mean that the plugin can't be used on py310 on any OS, as it hardcodes the ciphers, which is bad.

Just disable it on py36 for now, so that the tests pass. Or add the `HTTPSession.mount()` calls to `Plugin._get_streams()`, so that plugin initialization doesn't create the adapter instance where the error gets raised. Then the tests will pass, too. What then happens during runtime on systems using py36 where the SECLEVEL=1 cipher preset can't be used for some reason doesn't matter, as py36 is officially EOL and soon to be dropped from Streamlink.

Reply:
OK.  Yes, I realised you will be dropping 3.6 soon, which is why I suggested waiting, but if you're happy with the runtime behaviour in Windows and 3.6 likely not working until that happens, then I'm OK with that.  It's does seem likely that almost all people using Streamlink under Windows will not be running it with 3.6 anyway, so all things considered it's probably not a much of an issue.